umee/leverage/params.go

```diff
@@ -0,0 +1,169 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"gopkg.in/yaml.v3"
+)
+
+var _ paramtypes.ParamSet = &Params{}
+
+var (
+	KeyCompleteLiquidationThreshold = []byte("CompleteLiquidationThreshold")
+	KeyMinimumCloseFactor           = []byte("MinimumCloseFactor")
+	KeyOracleRewardFactor           = []byte("OracleRewardFactor")
+	KeySmallLiquidationSize         = []byte("SmallLiquidationSize")
+	KeyDirectLiquidationFee         = []byte("DirectLiquidationFee")
+)
+
+// ParamSetPairs implements the ParamSet interface and returns all the key/value
+// pairs pairs of x/leverage module's parameters.
+func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
+	return paramtypes.ParamSetPairs{
+		paramtypes.NewParamSetPair(
+			KeyCompleteLiquidationThreshold,
+			&p.CompleteLiquidationThreshold,
+			validateLiquidationThreshold,
+		),
+		paramtypes.NewParamSetPair(
+			KeyMinimumCloseFactor,
+			&p.MinimumCloseFactor,
+			validateMinimumCloseFactor,
+		),
+		paramtypes.NewParamSetPair(
+			KeyOracleRewardFactor,
+			&p.OracleRewardFactor,
+			validateOracleRewardFactor,
+		),
+		paramtypes.NewParamSetPair(
+			KeySmallLiquidationSize,
+			&p.SmallLiquidationSize,
+			validateSmallLiquidationSize,
+		),
+		paramtypes.NewParamSetPair(
+			KeyDirectLiquidationFee,
+			&p.DirectLiquidationFee,
+			validateDirectLiquidationFee,
+		),
+	}
+}
+
+// String implements the Stringer interface.
+func (p Params) String() string {
+	out, _ := yaml.Marshal(p)
+	return string(out)
+}
+
+// ParamKeyTable returns the x/leverage module's parameter KeyTable expected by
+// the x/params module.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// DefaultParams returns a default set of parameters.
+func DefaultParams() Params {
+	return Params{
+		CompleteLiquidationThreshold: sdk.MustNewDecFromStr("0.4"),
+		MinimumCloseFactor:           sdk.MustNewDecFromStr("0.05"),
+		OracleRewardFactor:           sdk.MustNewDecFromStr("0.01"),
+		SmallLiquidationSize:         sdk.MustNewDecFromStr("500.00"),
+		DirectLiquidationFee:         sdk.MustNewDecFromStr("0.05"),
+	}
```

Comment:
any significance for this values, or they are from Umee?

Reply:
they are from umee, added them to get rid of errors in the pb files, needed the pb files for xcclookup

Reply:
no significance, this was just required to remove errors from the pb files that I had to add - they are used by xcclookup